<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="6641" started-at="2016-08-03T20:47:54Z" finished-at="2016-08-03T20:48:00Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="6641" started-at="2016-08-03T20:47:54Z" finished-at="2016-08-03T20:48:00Z">
      <class name="groovy.NewTest">
        <test-method status="PASS" signature="beforeTest()[pri:0, instance:groovy.NewTest@5d8c1b8]" name="beforeTest" is-config="true" duration-ms="5545" started-at="2016-08-04T02:17:54Z" finished-at="2016-08-04T02:17:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method status="FAIL" signature="dayTest()[pri:0, instance:groovy.NewTest@5d8c1b8]" name="dayTest" duration-ms="755" started-at="2016-08-04T02:17:59Z" finished-at="2016-08-04T02:18:00Z">
          <exception class="org.codehaus.groovy.runtime.powerassert.PowerAssertionError">
            <message>
              <![CDATA[assert utill.returnLocator(str)==new Date().day
       |     |             |   | |          |
       |     4             |   | |          4
       |                   |   | Thu Aug 04 02:18:00 IST 2016
       |                   |   false
       |                   Thursday
       groovy.Utility@3c4d63d0]]>
            </message>
            <full-stacktrace>
              <![CDATA[Assertion failed: 

assert utill.returnLocator(str)==new Date().day
       |     |             |   | |          |
       |     4             |   | |          4
       |                   |   | Thu Aug 04 02:18:00 IST 2016
       |                   |   false
       |                   Thursday
       groovy.Utility@3c4d63d0

	at org.codehaus.groovy.runtime.InvokerHelper.assertFailed(InvokerHelper.java:398)
	at org.codehaus.groovy.runtime.ScriptBytecodeAdapter.assertFailed(ScriptBytecodeAdapter.java:648)
	at groovy.NewTest$_dayTest_closure1.doCall(NewTest.groovy:24)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:90)
	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:324)
	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:292)
	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1015)
	at org.codehaus.groovy.runtime.callsite.PogoMetaClassSite.call(PogoMetaClassSite.java:39)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:120)
	at groovy.Keyword.getInfo(Keyword.groovy:9)
	at groovy.Keyword$getInfo$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:45)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:108)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:116)
	at groovy.NewTest.dayTest(NewTest.groovy:27)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:646)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:823)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1131)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:778)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1225)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1150)
	at org.testng.TestNG.runSuites(TestNG.java:1075)
	at org.testng.TestNG.run(TestNG.java:1047)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:126)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:137)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:58)
]]>
            </full-stacktrace>
          </exception> <!-- org.codehaus.groovy.runtime.powerassert.PowerAssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- dayTest -->
        <test-method status="PASS" signature="monthTest()[pri:0, instance:groovy.NewTest@5d8c1b8]" name="monthTest" duration-ms="154" started-at="2016-08-04T02:18:00Z" finished-at="2016-08-04T02:18:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- monthTest -->
        <test-method status="PASS" signature="yearTest()[pri:0, instance:groovy.NewTest@5d8c1b8]" name="yearTest" duration-ms="144" started-at="2016-08-04T02:18:00Z" finished-at="2016-08-04T02:18:00Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- yearTest -->
      </class> <!-- groovy.NewTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
